webapp/src/containers/createIncident/index.tsx

```diff
@@ -54,6 +53,7 @@ const CreateIncident = () => {
 
     // Reset the field states
     const resetFieldStates = useCallback(() => {
+        setSubscriptionPayload(null);
```

Comment:
Re-ordering of this line wasn't necessary. We should avoid unnecessary changes.

Reply:
THis change was made because if we update the subscription payload later then we the showResultPanel state will be reset to true and we will see a result panel the next time we run the slash command